pkg/upload/android-proguard.go

```diff
@@ -27,6 +27,10 @@ func ProcessAndroidProguard(apiKey string, applicationId string, appManifestPath
 	var appManifestPathExpected string
 	var err error
 
+	if dryRun {
+		log.Info("Performing dry run - no files will be uploaded")
+	}
```

Comment:
Does dryRun now apply to all commands? Could we just put a message at the very start, top-level? I guess it just needs to be worded more generally – "no data will be sent to BugSnag"?